packages/expo-local-authentication/src/LocalAuthentication.js

```diff
@@ -47,6 +45,6 @@ export async function authenticateAsync(
   }
 }
 
-export function cancelAuthenticate(): void {
-  LocalAuthentication.cancelAuthenticate();
+export function cancelAuthenticate(): Promise<void> {
+  return LocalAuthentication.cancelAuthenticate();
```

Comment:
On Android this will return `Promise<null>` which is pointless over `Promise<void>` here, just write `await LocalAuthentication.cancelAuthenticate();` with no return value. (Also heads up, iOS looks to be missing cancelAuthenticate natively.)